mod-orgs/schemas/agreement.json

```diff
@@ -17,7 +17,7 @@
       "description": "The discount amount of this agreement",
```

Comment:
Could you please fix doubled `description` for `id` property above?

Reply:
Do we even need an id after we removed the API for agreements?